src/helpers/Svg/Svg.js

```diff
@@ -78,8 +78,8 @@ export const isInternalContentURL = (url) => {
 
 export const flattenToContentURL = (url) => {
   url = url ? flattenToAppURL(url) : url;
-  if (url?.startsWith('http')) {
-    if (isInternalContentURL(url)) {
+  if (url?.startsWith('http') || url?.startsWith('https')) {
```

Comment:
@nileshgulia1 

1. Redundant as `startsWith('http')` includes also `https`
2. I think `if (isInternalContentURL(url)) {` is the right condition we're looking for